src/components/Dashboard/Dashboard.story.jsx

```diff
@@ -618,6 +689,42 @@ storiesOf('Watson IoT Experimental|Dashboard', module)
       </FullWidthWrapper>
     );
   })
+  .add('full screen table card', () => {
```

Comment:
When the table is scrolled, the pagination sticks and overlays rows
![image](https://user-images.githubusercontent.com/3360588/66329394-0b413d80-e8f4-11e9-82a7-ff4565c0036d.png)

Reply:
fixed this now